upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = libsrtp2-2.2.0
```

Comment:
```
$ tar tf v2.2.0.tar.gz 
libsrtp-2.2.0/
```

The directory is called `libsrtp-2.2.0`.

Reply:
...and fixed.